django_loci/tests/__init__.py

```diff
@@ -94,6 +94,20 @@ def _login_as_admin(self):
         self.client.force_login(admin)
         return admin
 
+    def _create_readonly_admin(self, **kwargs):
+        """Creates a read-only admin user with view permissions for the specified models."""
+        #
+        from django.contrib.auth.models import Permission
```

Comment:
Do we need this import here or can it be moved to top?

Reply:
I was getting `Apps aren't loaded yet.`. I think it is because during test discovery, apps are not loaded.

Reply:
Would be good to add a comment for that for future references

Reply:
Comment Added